app/connectors/SchemeClient.scala

```diff
@@ -17,75 +17,46 @@
 package connectors
 
 import config.CSRHttp
-import connectors.SchemeClient.CannotFindSelection
-import models.UniqueIdentifier
-import models.frameworks.{ Alternatives, LocationAndSchemeSelection, Preference, Region }
+import connectors.SchemeClient.{CannotUpdateSchemePreferences, SchemePreferencesNotFound}
+import models.{SelectedSchemes, UniqueIdentifier}
 import play.api.http.Status._
-import play.api.libs.json.Json
-import uk.gov.hmrc.play.http.{ HeaderCarrier, HttpResponse, NotFoundException }
+import uk.gov.hmrc.play.http.{BadRequestException, HeaderCarrier, HttpResponse, NotFoundException}
 
 import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
 
 trait SchemeClient {
 
   import config.FrontendAppConfig.faststreamConfig._
 
   val http: CSRHttp
 
-  def getAvailableFrameworksWithLocations(applicationId: UniqueIdentifier)(implicit hc: HeaderCarrier): Future[List[Region]] =
-    http.GET(s"${url.host}${url.base}/frameworks-available-to-application/$applicationId").map(_.json.as[List[Region]])
-
-  def getSelection(applicationId: UniqueIdentifier)(implicit hc: HeaderCarrier): Future[LocationAndSchemeSelection] =
-    http.GET(s"${url.host}${url.base}/framework-preference/$applicationId").map(_.json.as[LocationAndSchemeSelection]).recover {
-      case e: NotFoundException => throw new CannotFindSelection()
+  def getSchemePreferences(applicationId: UniqueIdentifier)(implicit hc: HeaderCarrier) = {
+    http.GET(
+      s"${url.host}${url.base}/scheme-preferences/$applicationId"
+    ).map(
+      httpResponse => httpResponse.json.as[SelectedSchemes]
```

Comment:
SchemePreferences has been chosen based on story description. https://jira.tools.tax.service.gov.uk/browse/FSET-598

At the moment we have different case classes SelectedSchemes and SchemePreferences to represent form and exchange respectively. Perhaps selectedSchemes might go away with the new changes.